tests/server/blueprints/cases/test_cases_controllers.py

```diff
@@ -109,45 +107,6 @@ def test_phenotype_genes_matching_phenotypes(gene_database, case_obj, hpo_term,
     assert len(pheno_dict["Analysed genes"]["genes"].split(", ")) == 3
 
 
-def test_coverage_stats(app, monkeypatch):
```

Comment:
Simply moved this test on a dedicated file containing tests for the chanjo extension